### DISCUSSION TITLE
Accessibility Tests

### DISCUSSION THREAD

**Discussion by Utsav-06** [destrex271/pgweb-testing-harness#29] [Q&A] [answered]
As you have mentioned that all the Functional Tests were completed. 

What Accessibility and Utility Tests are needed in this project? 
Please provide some info.

**Comment by destrex271** [comment, marked as answer]
Hi Utsav! Thanks for your interest in the project. Currently Accessibility tests are being done using unlighthouse but we are generating a lot of html reports for each page which becomes very tedious to work with. 
We want a way to extract all the accessibility related info from those reports compiled in a single place.
